perf/lua/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+set(TARANTOOL_BIN $<TARGET_FILE:tarantool>)
+set(RUN_PERF_LUA_TEST "")
+
+function(create_perf_lua_test)
+  cmake_parse_arguments(
```

Comment:
Minor: I'd rather `cmake_parse_arguments` has way too complex signature, so I recommend to use as close as you can to this [snippet](https://github.com/tarantool/luajit/blob/tarantool/master/cmake/MakeSourceList.cmake#L24-L37). Anyway, this is quite minor nit, so feel free to ignore, if you have any strong arguments for your approach.

Reply:
I have no objections, this looks more clear.

```diff
diff --git a/perf/CMakeLists.txt b/perf/CMakeLists.txt
index 63108c04d8..4c6a619fd0 100644
--- a/perf/CMakeLists.txt
+++ b/perf/CMakeLists.txt
@@ -25,13 +25,18 @@ include_directories(${EXTRA_CORE_INCLUDE_DIRS})
 set(RUN_PERF_C_TESTS_LIST "")
 
 function(create_perf_test_target)
-  cmake_parse_arguments(
-    PERF
-    ""
-    "TARGET"
-    ""
-    ${ARGN}
-  )
+  set(prefix PERF)
+  set(noValues)
+  set(singleValues TARGET)
+  set(multiValues)
+
+  # FIXME: if we update to CMake >= 3.5, can remove this line.
+  include(CMakeParseArguments)
+  cmake_parse_arguments(${prefix}
+                        "${noValues}"
+                        "${singleValues}"
+                        "${multiValues}"
+                        ${ARGN})
   message(STATUS "Creating C performance test ${PERF_TARGET}_perftest")
   add_custom_target(${PERF_TARGET}_perftest
                     COMMAND "$<TARGET_FILE:${PERF_TARGET}.perftest>"
@@ -42,13 +47,18 @@ function(create_perf_test_target)
 endfunction()
 
 function(create_perf_test)
-  cmake_parse_arguments(
-    PERF
-    ""
-    "NAME"
-    "SOURCES;LIBRARIES"
-    ${ARGN}
-  )
+  set(prefix PERF)
+  set(noValues)
+  set(singleValues NAME)
+  set(multiValues "SOURCES;LIBRARIES")
+
+  # FIXME: if we update to CMake >= 3.5, can remove this line.
+  include(CMakeParseArguments)
+  cmake_parse_arguments(${prefix}
+                        "${noValues}"
+                        "${singleValues}"
+                        "${multiValues}"
+                        ${ARGN})
   add_executable(${PERF_TARGET}.perftest ${PERF_SOURCES})
   target_link_libraries(${PERF_TARGET}.perftest PUBLIC ${PERF_LIBRARIES})
 endfunction()
diff --git a/perf/lua/CMakeLists.txt b/perf/lua/CMakeLists.txt
index 908b293efd..e33aaeaea6 100644
--- a/perf/lua/CMakeLists.txt
+++ b/perf/lua/CMakeLists.txt
@@ -2,13 +2,19 @@ set(TARANTOOL_BIN $<TARGET_FILE:tarantool>)
 set(RUN_PERF_LUA_TESTS_LIST "")
 
 function(create_perf_lua_test)
-  cmake_parse_arguments(
-    PERF
-    ""
-    "NAME"
-    ""
-    ${ARGN}
-  )
+  set(prefix PERF)
+  set(noValues)
+  set(singleValues NAME)
+  set(multiValues)
+
+  # FIXME: if we update to CMake >= 3.5, can remove this line.
+  include(CMakeParseArguments)
+  cmake_parse_arguments(${prefix}
+                        "${noValues}"
+                        "${singleValues}"
+                        "${multiValues}"
+                        ${ARGN})
+
   message(STATUS "Creating Lua performance test ${PERF_NAME}_perftest")
   set(TEST_PATH ${CMAKE_CURRENT_SOURCE_DIR}/${PERF_NAME}.lua)
   add_custom_target(${PERF_NAME}_perftest
```